### PR TITLE
!!!SEIZE POWER HIJACK AMEND AUTOCRACY PR!!!

### PR DESCRIPTION
**Pull request by hongaar**
- Implemented `__all__` in github_api module (fixes PyCharms 'Cannot find reference 'xxx' in xxx.py', see https://stackoverflow.com/questions/23248017/cannot-find-reference-xxx-in-init-py-python-pycharm).

### CODE REVIEW

**Comment by hongaar** [comment]
I'm actually not sure how to properly do this in Python. Is it okay to do it like this?

**Comment by reddraggone9** [comment]
[`__all__` should be a list of strings](https://docs.python.org/3.6/tutorial/modules.html#importing-from-a-package).

File: github_api/__init__.py [hunk 1]
```diff
@@ -25,7 +36,7 @@ class API(object):
 
     BASE_URL = "https://api.github.com"
     BASE_HEADERS = {
-        #"Accept": "application/vnd.github.v3+json"
+        # "Accept": "application/vnd.github.v3+json"
```
Comment (PlasmaPower) [inline github_api/__init__.py]: Already fixed on master.

**Comment by hongaar** [comment]
@reddraggone9 still not sure how to implement this, but it works now in PyCharm (although simply importing submodules is enough to make autocomplete work, `__all__` is not strictly required but doesn't harm either?).

**Comment by hongaar** [comment]
maybe it get some attention now to get last votes 😉

**Comment by Syt-ha** [comment]
#clickbait

**Comment by mdcfe** [comment]
When the gitignore is the reason you have a conflict

**Comment by phil-r** [comment]
That's actually cool that I had access to @hongaar repo. I could fix the conflict while he was away ;)

**Pull request merged**